src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -58,11 +63,14 @@ def load_command_table(self, _):
         g.custom_command('repo update', 'config_repo_update')
         g.custom_command('repo list', 'config_repo_list')
 
-    with self.command_group('spring-cloud app', client_factory=cf_spring_cloud_20220101preview,
+    with self.command_group('spring-cloud app', custom_command_type=app_command,
```

Comment:
The 4 commands' traffic will be routing to the app.py
- `spring-cloud app create`
- `spring-cloud app update`
- `spring-cloud app deploy`
- `spring-cloud app deployment create`